src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/exceptions/NoPdfFileFoundException.java

```diff
@@ -12,8 +12,8 @@
 
     private final String zipFileName;
 
-    public NoPdfFileFoundException(String container, String zipFileName) {
-        super("Zip file '" + zipFileName + "' does not contain pdf files");
+    public NoPdfFileFoundException(String container, String zipFileName, String fileName) {
+        super("Zip '" + zipFileName + "' contains non-pdf file: " + fileName);
```

Comment:
This changes the meaning of this exception completely.
Previously: there are **no** pdf files at all.
Now: there may be some pdfs but also a file that is not a pdf.

Is this indented?